### ISSUE TITLE
Build fails on GitHub Actions

### ISSUE THREAD

**Issue by PenguinDOOM** [kcat/openal-soft#449]
I'm not sure why, but it fails to build on GitHub Actions.
log
[logs_9.zip](https://github.com/kcat/openal-soft/files/4876495/logs_9.zip)

[workflow file](https://github.com/PenguinDOOM/openal-soft/actions/runs/158792544/workflow)

**Comment by kcat** [comment]
By the looks of it, MSVC is failing on its own header:
```
  error.cpp
C:\Program Files (x86)\Windows Kits\10\Include\10.0.17763.0\um\winuser.h(14179): error C2143: syntax error: missing ';' before '__cdecl' [D:\a\openal-soft\openal-soft\build\soft_oal.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.17763.0\um\winuser.h(14179): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [D:\a\openal-soft\openal-soft\build\soft_oal.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.17763.0\um\winuser.h(14181): error C2146: syntax error: missing ')' before identifier 'value' [D:\a\openal-soft\openal-soft\build\soft_oal.vcxproj]
```
`winuser.h` is being included by something, which gets confused thinking a declaration is made before  `__cdecl`, and the supposed declaration is missing a type. That's typically a result of a macro or type alias not being defined when it should be, but I have no idea why that would happen here. It only happens with some sources, and OpenAL Soft's not doing anything special to include headers.

**Comment by Hoikas** [comment]
This also happens on [AppVeyor](https://ci.appveyor.com/project/Hoikas/plasmaprefix/builds/33966096).

**Comment by kcat** [comment]
This should be fixed now with commit 13698362f1726326ab60180b04a86df79b518614, assuming it's the same error as in #498.
